src/NexusMods.App.UI/Pages/Downloads/InProgressViewModel.cs

```diff
@@ -89,6 +90,11 @@ public class InProgressViewModel : APageViewModel<IInProgressViewModel>, IInProg
 
         this.WhenActivated(d =>
         {
+            if (WindowManager.TryGetActiveWindow(out var window))
+            {
+                window.WorkspaceController.SetTabTitle(Language.InProgressDownloadsPage_Title, WorkspaceId, PanelId, TabId);
+            }
```

Comment:
This can be replaced with

```csharp
GetWorkspaceController().SetTabTitle(Language.InProgressDownloadsPage_Title, WorkspaceId, PanelId, TabId);
```